docs/en/trust.rst

```diff
@@ -536,16 +537,16 @@ In offline flows, Trust Chain verification enables the assessment of the reliabi
 Wallet Instance Attestation
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The Wallet Provider issues a Wallet Instance Attestation, certifying the operational status of its Wallet Instances, including one of their public keys. 
+The Wallet Provider issues the Wallet Instance Attestation, certifying the operational status of its Wallet Instances and including one of their public keys. 
 
 The Wallet Instance Attestation contains the Trust Chain that attests the validity for its issuer (Wallet Provider).
 
-The Wallet Instance presents its Wallet Instance Attestation within the signed request during the PID issuance phase, containing the Trust Chain related to the Wallet Provider. The PID Provider issues a PID for the public key contained in the Wallet Instance Attestation, producing the Holder Key Binding within the issued PID.
+The Wallet Instance presents its Wallet Instance Attestation within the signed request during the PID issuance phase, containing the Trust Chain related to the Wallet Provider. The PID Provider issues a PID including the public key contained in the Wallet Instance Attestation, this produces the Holder Key Binding within the issued PID.
```

Comment:
The Holder Key Binding is actually not done with the public key contained in the WIA.

Reply:
yes, deprecation here

Reply:
done